quiz-app/src/assets/translations/hin.json

```diff
@@ -0,0 +1,1017 @@
+							{
+								"answerText": "False",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "True",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "IoT हब के नाम अद्वितीय होने चाहिए:",
+						"answerOptions": [
+							{
+								"answerText": "True",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "false"
+							}
+						]
+					}
+				]
+			},
+			{
+				"id": 17,
+				"title": "पाठ 9 - बादल के लिए अपने आवेदन तर्क माइग्रेट: प्री लेक्चर प्रश्नोत्तरी",
+				"quiz": [
+					{
+						"questionText": "आप IoT घटनाओं का जवाब देने के लिए सर्वरलेस कोड का उपयोग कर सकते हैं",
+						"answerOptions": [
+							{
+								"answerText": "True",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "जब आप IoT ईवेंट को IoT हब भेजते हैं:",
+						"answerOptions": [
+							{
+								"answerText": "केवल एक सेवा IoT हब से घटनाओं को पढ़ सकते है",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "सेवाओं की किसी भी संख्या IoT हब से घटनाओं को पढ़ सकते है",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "सेवाएं IoT हब से घटनाओं को नहीं पढ़ सकते हैं, उन्हें सीधे डिवाइस से कनेक्ट करना होगा",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "IoT हब से घटनाओं को पढ़ते समय, आप केवल क्लाउड में चलने वाले कोड का उपयोग करके ऐसा कर सकते हैं",
+						"answerOptions": [
+							{
+								"answerText": "True",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "true"
+							}
+						]
+					}
+				]
+			},
+			{
+				"id": 18,
+				"title": "पाठ 9 - बादल के लिए अपने आवेदन तर्क माइग्रेट: पोस्ट-लेक्चर प्रश्नोत्तरी",
+				"quiz": [
+					{
+						"questionText": "Azure फ़ंक्शंस को स्थानीय रूप से चलाया और डिबग किया जा सकता है:",
+						"answerOptions": [
+							{
+								"answerText": "True",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "False",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "सर्वरलेस कोड केवल जावास्क्रिप्ट और COBOL में लिखा जा सकता है:",
+						"answerOptions": [
+							{
+								"answerText": "False",
+								"isCorrect": "true"
+							},
+							{
+								"answerText": "True",
+								"isCorrect": "false"
+							}
+						]
+					},
+					{
+						"questionText": "क्लाउड पर फ़ंक्शन ऐप की तैनाती करते समय, आपको बनाने और तैनात करने की आवश्यकता है:",
+						"answerOptions": [
+							{
+								"answerText": "केवल एक फ़ंक्शन ऐप",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "एक कार्य ऐप और केवल एक स्टोरेज खाता",
+								"isCorrect": "false"
+							},
+							{
+								"answerText": "एक फ़ंक्शन ऐप, एक स्टोरेज अकाउंट और एप्लिकेशन सेटिंग्स",
+								"isCorrect": "frue"
```

Comment:
"answerText": "एक फ़ंक्शन ऐप, एक स्टोरेज अकाउंट और एप्लिकेशन सेटिंग्स",
								"isCorrect": "frue"

change it to true.